core/lib/protobuf_config/src/proto/config/snapshot_recovery.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package zksync.config.snapshot_recovery;
+
+message SnapshotRecovery {
+  optional bool enabled = 1;
+  optional uint64 postgres_max_concurrency = 2;
+  optional uint64 tree_chunk_size = 3;
+  optional uint32 l1_batch = 4;
+  optional uint64 tree_parallel_persistence_buffer = 5;
```

Comment:
Bikeshedding: Abstractly, I'd prefer `postgres` and `tree` to be extracted to separate sub-config objects so that the config is more structured:

```yaml
snapshot_recovery:
  enabled: true
  l1_batch: 9112
  postgres:
    max_concurrency: 10
  tree:
    chunk_size: 200000
    parallel_persistence_buffer: 4
```

Not a strong preference though, so feel free to leave as-is if you disagree.